### ISSUE TITLE
Broken Cypress test: `Spaces > should render subspaces in the space panel only when expanded`

### ISSUE THREAD

**Issue by richvdh** [vector-im/element-web#26607]
```
cypress/e2e/spaces/spaces.spec.ts:
...
  1) Spaces
       should render subspaces in the space panel only when expanded:
     AssertionError: 1 accessibility violation was detected: expected 1 to equal 0
      at Context.eval (webpack://matrix-react-sdk/./node_modules/cypress-axe/dist/index.js:100:0)
```

**Comment by richvdh** [comment]
@MidhunSureshR suggests this is an actual longstanding problem, which is only being detected now because, until recently, the tests were being run on Electron and are now being run on Chrome (?)

**Comment by t3chguy** [comment]
![image](https://github.com/vector-im/element-web/assets/2403652/c1be6b87-27fb-4b90-ac8e-17c0e726a6f0)
